src/quote/Quote.ts

```diff
@@ -108,3 +117,269 @@ export function createAgreedQuote(
     claimerSignature: signature,
   }
 }
+/**
```

Comment:
I am not sure about the checks within the `Quote`.

Reply:
Yeah, the quotes produce a lot of overhead code. But let's keep it for now and refactor later